plugin/pkg/scheduler/algorithm/priorities/interpod_affinity.go

```diff
@@ -97,6 +97,7 @@ func (p *podAffinityPriorityMap) processTerm(term *api.PodAffinityTerm, podDefin
 				if p.failureDomains.NodesHaveSameTopologyKey(node, fixedNode, term.TopologyKey) {
 					p.counts[node.Name] += weight
 				}
+				// TODO i think we need to  add processing logic for pod labels here?
```

Comment:
ahhh i see whats going on.  the region stuff isnt in the labels, its separate i think.  /me digs more